pkg/testers/node/node.go

```diff
@@ -267,6 +268,7 @@ func (t *Tester) constructArgs() []string {
 		"USE_DOCKERIZED_BUILD=" + strconv.FormatBool(t.UseDockerizedBuild),
 		"TARGET_BUILD_ARCH=" + t.TargetBuildArch,
 		"TIMEOUT=" + t.Timeout.String(),
+		"LABEL_FILTER= " + t.LabelFilter,
```

Comment:
```suggestion
		"LABEL_FILTER=" + t.LabelFilter,
```

you have a whitespace there